index.bs

```diff
@@ -352,6 +352,38 @@ An attribution source is a [=struct=] with the following items:
 :: An [=attribution filter map=].
 : <dfn>debug key</dfn>
 :: Null or a non-negative 64-bit integer.
+: aggregatable source
```

Comment:
```suggestion
: <dfn>aggregatable source</dfn>
```

Reply:
Will add this when adding histogram creation algorithm.